styles/style.css

```diff
@@ -0,0 +1,84 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  top: 20px;
+  font-family: Poppins, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  font-weight: 700;
+  padding: 20px;
+  color: #6070ff;
+}
+
+.menu-btn {
+  color: #6070ff;
+  padding: 20px;
+  width: 70px;
+}
+
+.headline-section {
+  background-image: url(../img/Header_bg.svg);
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: 110%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  padding: 150px 50px;
+  align-items: center;
+}
+
+.headline-header {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+  width: 320px;
```

Comment:
- [ ] Please make sure that you completely follow the design guidelines. If you take a look at the two images down below you'll find a slight difference between your design and the original one. Kindly fix this issue.
Hint: the two sections (Toolbar and Headline) have two different paddings and fixed widths.
[Original Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
![Screenshot](https://user-images.githubusercontent.com/47625407/120523040-be1db380-c3d5-11eb-8002-70be43532113.png)
Your Design
![Screenshot1](https://user-images.githubusercontent.com/47625407/120523167-e60d1700-c3d5-11eb-8524-4b91875a66df.png)